packages/create-miniapp-cli/template/fullstack/pages/_app.tsx

```
@@ -0,0 +1,103 @@
+import { gql } from 'graphql-tag'
+import get from 'lodash/get'
+import getConfig from 'next/config'
+import Head from 'next/head'
+import React, { useContext } from 'react'
+
+import { extractReqLocale } from '@open-condo/locales/extractReqLocale'
+import { withApollo } from '@open-condo/next/apollo'
+import { withAuth } from '@open-condo/next/auth'
+import { useIntl, withIntl, LocaleContext } from '@open-condo/next/intl'
+
+import { AppFrameWrapper } from '@billing-connector/domains/common/components/containers/AppFrameWrapper'
```

ESLint

[error] '@billing-connector/domains/common/components/containers/AppFrameWrapper' import is restricted from being used by a pattern.

(no-restricted-imports)